### PR TITLE
Switch from local copies of registry files to use Vulkan-Headers

### PR DESCRIPTION
**Pull request by mark-lunarg**
This was hiding issues in the mock ICD due to the 75 header update.  Fixed that issue, renamed some files and moved to using vulkan registry files from the specified Vulkan-Headers repository.

### CODE REVIEW

File: icd/CMakeLists.txt [hunk 1]
```diff
@@ -9,8 +9,8 @@ set (PYTHON_CMD ${PYTHON_EXECUTABLE})
 # Define macro used for building vk.xml generated files
 macro(run_vk_xml_generate dependency output)
     add_custom_command(OUTPUT ${output}
-    COMMAND ${PYTHON_CMD} ${SCRIPTS_DIR}/lvl_genvk.py -registry ${SCRIPTS_DIR}/vk.xml ${output}
-    DEPENDS ${SCRIPTS_DIR}/vk.xml ${SCRIPTS_DIR}/generator.py ${SCRIPTS_DIR}/${dependency} ${SCRIPTS_DIR}/lvl_genvk.py ${SCRIPTS_DIR}/reg.py
+    COMMAND ${PYTHON_CMD} ${SCRIPTS_DIR}/v_t_genvk.py -registry ${SCRIPTS_DIR}/vk.xml ${output}
```
Comment (tobine) [inline icd/CMakeLists.txt]: Are we sold on v_t_genvk.py, or is vt_genvk.py ok? Reduces required shifts by 50%.

Comment (mark-lunarg) [reply icd/CMakeLists.txt]: I started off that way, but what should we then call the one in VulkanTools?

Comment (tobine) [reply icd/CMakeLists.txt]: This one could be "kvt_genvk.py" and that one could be "lvt_genvk.py"?

Comment (mark-lunarg) [reply icd/CMakeLists.txt]: OK Tobin, it shall be so.  Though, I have to say this is a lot coming from the creator of ~^~ field separators. 👍

Comment (tobine) [reply icd/CMakeLists.txt]: I seriously almost qualified my suggestion by saying "just remember this is coming from the creator of the ~^~ delimiter". I know how much everyone loves that!

**Comment by mark-lunarg** [comment]
Updated script name and pushed to Master.

**Pull request closed without merging**